bin/rucio

```diff
@@ -2139,6 +2151,7 @@ def info_rule(args):
         print "Purge replicas:             %s" % rule['purge_replicas']
         print "Notification:               %s" % rule['notification']
         print "End of life:                %s" % rule['eol_at']
+        print "Child rule Id:              %s" % rule['child_rule_id']
```

Comment:
capital R in rule or lowercase I in id just to be consistent ;-)

Reply:
🙄  will fix it 😄